### PR TITLE
Fixed "created" Metadata not defaulting to meeting start time

### LINKED ISSUE #10: Creation dates should be the meeting date
Right now, DC created is not set. It should be set to the recording date.

### PR DESCRIPTION
**Pull request by Arnei**
Resolves #10.

### CODE REVIEW

File: post-archive/post_archive.rb [hunk 1]
```diff
@@ -658,8 +658,8 @@ def cleanup(tmp_path, meeting_id)
 BigBlueButton.logger.info( tracks)
 
 # Create metadata file dublincore
-dc_data = OcDublincore::parseDcMetadata(meeting_metadata, server: $oc_server, user: $oc_user, password: $oc_password)
-dublincore = OcDublincore::createDublincore(dc_data)
+dc_data = OcDublincore::parseDcMetadata(meeting_metadata, startTime: real_start_time, endTime: real_end_time,
+                                        server: $oc_server, user: $oc_user, password: $oc_password)dublincore = OcDublincore::createDublincore(dc_data)
```
Comment (Rillke) [inline post-archive/post_archive.rb:662]:
```suggestion
                                        server: $oc_server, user: $oc_user, password: $oc_password)
dublincore = OcDublincore::createDublincore(dc_data)
```

**Review by Rillke — CHANGES_REQUESTED** [review]
Syntax error: Each expression are delimited by semicolons(;) or newlines.

**Comment by Rillke** [comment]
Hi @Arnei do you know why even after applying this patch, the stopped time is not displayed in the opencast admin UI?
Screenshot from 2021-05-09 11-49-11
Screenshot from 2021-05-09 11-49-17

File: post-archive/post_archive.rb [hunk 2]
```diff
@@ -658,8 +658,8 @@ def cleanup(tmp_path, meeting_id)
 BigBlueButton.logger.info( tracks)
 
 # Create metadata file dublincore
-dc_data = OcDublincore::parseDcMetadata(meeting_metadata, server: $oc_server, user: $oc_user, password: $oc_password)
-dublincore = OcDublincore::createDublincore(dc_data)
+dc_data = OcDublincore::parseDcMetadata(meeting_metadata, startTime: real_start_time, endTime: real_end_time,
```
Comment (Rillke) [inline post-archive/post_archive.rb:661]:
```suggestion
dc_data = OcDublincore::parseDcMetadata(meeting_metadata, startTime: real_start_time, stopTime: real_end_time,
```

**Review by Rillke — CHANGES_REQUESTED** [review]
`parseDcMetadata` [expects args[:stopTime]](https://github.com/elan-ev/opencast-bigbluebutton-integration/blob/master/oc_modules/oc_dublincore.rb#L104)

**Comment by Arnei** [comment]
Hi @Rillke, thanks for all the reviews! Regarding your question:

> Hi @Arnei do you know why even after applying this patch, the stopped time is not displayed in the opencast admin UI?

As far as I can tell, this problem started occuring with Opencast 9. I created a PR that should fix it https://github.com/elan-ev/opencast-bigbluebutton-integration/pull/31.

**Comment by Rillke** [comment]
> As far as I can tell, this problem started occuring with Opencast 9. I created a PR that should fix it #31.

Oh, we have Opencast 8; right after the `s/end/stop/`Time fix, the dates and duration appeared correctly. But it's quite possible that Opencast 9 needs that temporal field. Thanks :)

**Pull request merged**